docs/testnet/onomy-testnet-docs/faucet.md

```diff
@@ -0,0 +1,105 @@
+# Onomy testnet faucet
+
+In order to use the faucet and receive testnet tokens you'll first need to setup a wallet
+
+## What do I need?
+
+A Linux server with any modern Linux distribution, 2gb of ram and at least 20gb storage. Requirements are very minimal.
+
+### Download/install Onomy chain binaries
+```
+To download binary follow these commands
+mkdir binaries
+cd binaries
+wget https://github.com/onomyprotocol/onomy/releases/download/v0.0.1/onomyd
+wget https://github.com/onomyprotocol/onomy/releases/download/v0.0.1/gbt
+wget https://github.com/onomyprotocol/onomy/releases/download/v0.0.1/geth
+cd ..
+chmod -R +x binaries
+export PATH=$PATH:$HOME/binaries/
+
+
+or If you have Fedora (Fedora 34) or Redhat (Red Hat Enterprise Linux 8.4 (Ootpa))
+ and you want to make binaries yourself, then follow these steps
+
+sudo yum install -y git
+git clone -b dev https://github.com/onomyprotocol/onomy.git
+cd onomy/deploy/testnet
+bash bin.sh
+```
+### Init the config files
+
+```
+cd $HOME
+onomyd --home $HOME/onomy/onomy init mymoniker --chain-id onomy
+```
+### Generate your key
+
+Be sure to back up the phrase you get! You’ll need it in a bit. If you don't back up the phrase here just follow the steps again to generate a new key.
+
+Note 'myvalidatorkeyname' is just the name of your key here, you can pick anything you like, just remember it later.
+
+You'll be prompted to create a password, I suggest you pick something short since you'll be typing it a lot
+
+```
+cd $HOME
+onomyd --home $HOME/onomy/onomy init mymoniker --chain-id onomy
```

Comment:
Is `$HOME/onomy/` correct? It's traditional within tendermint projects to do `$HOME/.onomy`. Not saying it's wrong, just clarifying. This applies to all the usages.

I'd recommend removing this line since it already exists above.

Reply:
all above suggestion has been implemented